src/apify_client/_utils.py

```diff
@@ -122,3 +125,51 @@ def _catch_not_found_or_throw(exc: ApifyApiError) -> None:
         raise exc
 
     return None
+
+
+def _encode_json_to_base64(data: JSONSerializable) -> bytes:
```

Comment:
Seems to be a misnomer - the function doesn't encode a JSON string, but a JSON-serializable types of objects. Same with the _decode_base64_json function, which also lacks "to" between base64 and json.

Reply:
not sure what you mean by this, can you please elaborate, ideally with an example_

Reply:
First of all, I think the _decode_base64_json should be _decode_base64_to_json.
However, my main complain is that _encode_json_to_base64 doesn't actually take JSON as an input, it usually takes a dictionary, doesn't it?

Reply:
Renamed...as for `_encode_json_to_base64`, it can take any `JSONSerializable` value, look at the doc tests in `_decode_base64_to_json` where this is featured